legal-api/report-templates/template-parts/common/statement.html

```diff
@@ -0,0 +1,26 @@
+{% if business.legalType == 'BEN' %}
+  <div class="no-page-break">
+    <div class="section-title mt-4">B.C. Benefit Company Statement</div>
+    <div class="section-data mt-4">
+        This company is a benefit company and, as such, has purposes that include conducting its
+        business in a responsible and sustainable manner and promoting one or more public benefits.
+    </div>
+  </div>
+{% elif business.legalType == 'CC' %}
+  <div class="no-page-break">
+    <div class="section-title mt-4">B.C. Community Contribution Company Statement</div>
+    <div class="section-data mt-4">
+      This company is a community contribution company, and, as such, has purposes beneficial to society.
+      This company is restricted, in accordance with Part 2.2 of the BCA, in its ability to pay dividends and to
+      distribute its assets on dissolution or otherwise.
+    </div>
+  </div>
+{% elif business.legalType == 'ULC' %}
+  <div class="no-page-break">
+    <div class="section-title mt-4">B.C. Unlimited Liability Company Statement</div>
+    <div class="section-data mt-4">
+      The shareholders of this company are jointly and severally liable to satisfy the debts and
+      liabilities of this company to the extent provided in section 51.3 of the Business Corporations Act.
+    </div>
+  </div>
+{% endif %}    
```

Comment:
Is there a case for LTD also?

Reply:
For now these are the statements available. We can extend it when we have more